Owl Carousel/Readme.md

```diff
@@ -0,0 +1,3 @@
+# Owl Carousel
+
```

Comment:
Please add a proper readme file like this: [example](https://github.com/mudit023/templates/blob/main/animated-newsletter/Readme.md)